sn_cli/src/subcommands/wallet.rs

```diff
@@ -39,6 +40,8 @@ pub enum WalletCmds {
     /// variable to a path you would prefer to work with.
     #[clap(verbatim_doc_comment)]
     Deposit,
+    /// Read a DBC from stdin and deposit it to the local wallet.
+    Read,
```

Comment:
Hey Anselme,

I'm not sure about adding this as another command in addition to `deposit`. The same functionality could be added to the `deposit` command.

Unless there is something distinctly different about this that I'm missing?

Reply:
I do like being able to read from stdin btw. We had that in the old CLI. However, I don't think it merits having another command.

Reply:
I agree, it should be Deposit. If I understand correctly, deposit reads files in a path provided as arg. Would it make sense to read from stdin if the path argument is not provided?

Reply:
Since the path wasn't an argument this is what I did:
```
# read from default path (unchanged)
safe wallet deposit

# read from stdin (new feature)
safe wallet deposit --stdin
```
And removed the `Read`
Good one @jacderida!